### ISSUE TITLE
"Open with" for folders are disabled since V1.16.2

### ISSUE THREAD

**Issue by bitfreak25** [mate-desktop/caja#734]
I've recently get a update on Debian Testing, where caja was updated from V1.16.1 to V1.16.2 . Yet I missed the option to right click on a folder and use the "open with ..." command.

I expect, it was introduced in [commit 921dbb418e1f7319d8d8100cc5cda4891e246c36](https://github.com/mate-desktop/caja/commit/921dbb418e1f7319d8d8100cc5cda4891e246c36). Especially in line 4790 from fm-directory-view.c . Could this be tested an fixed?

**Comment by monsta** [comment]
@lukefromdc: oops...

**Comment by lukefromdc** [comment]
I am trying to catch back up after four days away from my desk, I will have to split up
the changes previously made to find the culprit

**Comment by lukefromdc** [comment]
Fixed with https://github.com/mate-desktop/caja/commit/d575ecad1297e3200df6370efd5d9ae6db12b96a

**Comment by monsta** [comment]
Thanks, it works now. Looks like we can remove the whole code block, it seems to do nothing now:

```
	for (node = selection; node != NULL; node = node->next) {

		file = CAJA_FILE (node->data);

		other_applications_visible;
	}
```

**Comment by lukefromdc** [comment]
https://github.com/mate-desktop/caja/commit/14b802e267e66f99db00fec1d2e6d262e5b7fb5a
Removes the unused code. Thanks for finding it. I am way behind and thus prioritizing problems with my own code until I can catch up.

**Comment by monsta** [comment]
Ok, the fix is also ported to 1.16 branch now, so it will be included in the next 1.16.x release.
